twitter/View Controllers/TimelineViewController.m

```diff
@@ -10,31 +10,51 @@
 #import "APIManager.h"
 #import "AppDelegate.h"
 #import "LoginViewController.h"
+#import "TweetCell.h"
+#import "Tweet.h"
+#import "UIImageView+AFNetworking.h"
 
-@interface TimelineViewController ()
+@interface TimelineViewController () <UITableViewDelegate,UITableViewDataSource>
+@property (weak, nonatomic) IBOutlet UITableView *twitterFeedTableView;
 - (IBAction)didTapLogout:(id)sender;
+@property (strong, nonatomic) NSMutableArray* arrayOfTweets;
+@property (strong,nonatomic)  UIRefreshControl *refreshControl;
 
 @end
 
 @implementation TimelineViewController
 
-- (void)viewDidLoad {
-    [super viewDidLoad];
-    
-    // Get timeline
-    [[APIManager shared] getHomeTimelineWithCompletion:^(NSArray *tweets, NSError *error) {
+- (void)fetchTweets {
+    [[APIManager shared] getHomeTimelineWithCompletion:^(NSMutableArray *tweets, NSError *error) {
         if (tweets) {
             NSLog(@"😎😎😎 Successfully loaded home timeline");
-            for (NSDictionary *dictionary in tweets) {
-                NSString *text = dictionary[@"text"];
-                NSLog(@"%@", text);
-            }
+            self.arrayOfTweets = tweets;
+            NSLog(@"Here is the arrray of tweets: %@", self.arrayOfTweets);
+            [self.twitterFeedTableView reloadData];
         } else {
             NSLog(@"😫😫😫 Error getting home timeline: %@", error.localizedDescription);
```

Comment:
Should you be displaying the error to the user (or some sort of message when it fails refreshing)?